helm/libero-reviewer/templates/deployment-client.yaml

```diff
@@ -33,6 +33,8 @@ spec:
               value: "https://{{ include "libero-reviewer.fullname" . }}.elifesciences.org"
             - name: CLIENT_API_PROXY_URL
               value: "https://{{ include "libero-reviewer.fullname" . }}-submission"
```

Comment:
how can these URLs be `https` if there is no certificate (no `.elifesciences.org`-based domain?)

Reply:
too much of a secure by default mentality i guess